authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/integration/RequireOtpNotFrozenHelper.java

```diff
@@ -0,0 +1,80 @@
+package io.mosip.authentication.common.service.integration;
```

Comment:
Why new class created?

Reply:
To resolve circular dependency moved code to new class.